etc/configure.py

```diff
@@ -0,0 +1,29 @@
+import subprocess
+import os
+import virtualenv
```

Comment:
Maybe we could have alphabetically sorted imports and according to the python style guide ?